src/providers/auth.ts

```diff
@@ -54,7 +54,7 @@ export const provider = 'google.firebase.auth';
 /** @hidden */
 export const service = 'firebaseauth.googleapis.com';
 
-/** Resource level options */
+/** @public Resource level options */
```

Comment:
visibility modifiers should be on their own line

Reply:
Done